gui/src/daemon/model.rs

```diff
@@ -64,7 +64,7 @@ impl SpendTx {
         secp: &secp256k1::Secp256k1<impl secp256k1::Verification>,
         network: Network,
     ) -> Self {
-        let max_vbytes = desc.unsigned_tx_max_vbytes(&psbt.unsigned_tx);
+        let max_vbytes = desc.unsigned_tx_max_vbytes(&psbt.unsigned_tx, false);
```

Comment:
@edouardparis is `SpendTx::new` used for both the recovery and the primary-path spends? If only for primary path i could use `true` here instead to avoid overestimating.

Reply:
Yes used by both: https://github.com/wizardsardine/liana/blob/master/gui/src/app/state/recovery.rs#L173
but I can create a SpendTx::new_recovery or add an arg to the function

Reply:
If you think it's worth doing, let's open an issue to track this.

Reply:
Could we check for any transaction inputs with the nSequence being a recovery locktime to determine whether the primary path should be used? (I borrowed from https://github.com/wizardsardine/liana/pull/1125#discussion_r1647565509 :slightly_smiling_face:)